src/ocaml.ml

```diff
@@ -0,0 +1,17 @@
+open Import
+
+type t = Ocaml_version.t
```

Comment:
do we really need this type? and this entire module for that matter. It is only used once so just inline it there.